CONTRIBUTING.md

```diff
@@ -38,23 +67,16 @@ We are currently managing payment via Upwork. If you'd like to be paid for your
     [gpg]
         program = gpg
     ```
-1. [Open a PR](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork). Be sure to fill in all the required information on the PR template.
-1. An Expensify engineer will be automatically assigned to review your PR
-1. You will need all checks to pass:
```

Comment:
I think these are still important to highlight, including the CLA link. Is there a reason we removed this?

Reply:
Oversight on my part - as I thought we captured the part that was important, and didn't deem CLA stuff as fitting the bill. Apologies 😄, fixing now